.gitignore

```diff
@@ -16,3 +16,4 @@ packages
 .directory
 Thumbs.db
 *.userprefs
+*.ffs_db
```

Comment:
Is the sync file always the same name?  (Google seems to suggest `sync.ffs_db`?)  If so, this should probably be the filename rather than a wildcard and extension.